src/MicrosoftTeams.ts

```diff
@@ -38,6 +38,18 @@ interface Window {
   onNativeMessage(evt: MessageEvent): void;
 }
 
+/**
+ * adding ctrl+P and cmd+P handler
+ */
+document.addEventListener("keydown", function(event: KeyboardEvent): void {
```

Comment:
@sachingupta, I think it would be better to only register this handler when printing is enabled both for performance reasons and also to avoid interfering with any similar logic the developer might already have on their page.

Reply:
sounds good